demo/lib/host.bash

```diff
@@ -234,6 +235,23 @@ host-wait-vm-ssh-server() {
     [ "$retries" == "$retries_left" ] || echo ""
 }
 
+host-wait-cloud-init() {
+    retries=60
+    retries_left=$retries
+    while ! $SSH ${VM_SSH_USER}@${VM_IP} -o ConnectTimeout=2 cloud-init status --wait 2>/dev/null; do
```

Comment:
Fix: on `distro=centos-7` running `cloud-init status` as a user fails to a permission issue. I'd suggest adding `sudo`. That works on all distros.

Nitpick: suggestion on style: maybe ConnectTimeout could be more clearly a SSH parameter.
```
$SSH ${VM_SSH_USER}@${VM_IP} -o ConnectTimeout=2 COMMAND
->
$SSH -o ConnectTimeout=2 ${VM_SSH_USER}@${VM_IP} COMMAND
```

Reply:
Updated according to those suggestions.